homeassistant/components/lightwave/sensor.py

```diff
@@ -0,0 +1,84 @@
+"""Support for LightwaveRF TRV - Associated Battery."""
+import logging
+
+from homeassistant.const import CONF_NAME
+from homeassistant.helpers.entity import Entity
+
+from . import CONF_SERIAL, LIGHTWAVE_LINK, LIGHTWAVE_TRV_PROXY, LIGHTWAVE_TRV_PROXY_PORT
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_platform(hass, config, async_add_entities, discovery_info=None):
+    """Find and return battery."""
+    if not discovery_info:
+        return
+
+    batt = []
+
+    lwlink = hass.data[LIGHTWAVE_LINK]
+    trv_proxy_ip = hass.data[LIGHTWAVE_TRV_PROXY]
+    trv_proxy_port = hass.data[LIGHTWAVE_TRV_PROXY_PORT]
+
+    for device_id, device_config in discovery_info.items():
+        name = device_config[CONF_NAME]
+        serial = device_config[CONF_SERIAL]
+        batt.append(
+            LightwaveBattery(
+                name, device_id, lwlink, serial, trv_proxy_ip, trv_proxy_port
+            )
+        )
+
+    async_add_entities(batt)
+
+
+class LightwaveBattery(Entity):
+    """Lightwave TRV Battery."""
+
+    def __init__(self, name, device_id, lwlink, serial, trv_proxy_ip, trv_proxy_port):
+        """Initialize the Lightwave Trv battery sensor."""
+        self._name = name
+        self._device_id = device_id
+        self._state = None
+        self._lwlink = lwlink
+        self._serial = serial
+        self._device_class = "battery"
```

Comment:
Please import this device class from homeassistant/const.py.

Reply:
Done